discord/components.py

```diff
@@ -95,6 +135,10 @@ def _raw_construct(cls: type[C], **kwargs) -> C:
     def to_dict(self) -> dict[str, Any]:
         raise NotImplementedError
 
+    def is_v2(self) -> bool:
```

Comment:
Rename `is_v2` to `is_components_v2`

Reply:
i feel like this naming is redundant for the component classes but idk

Reply:
ok that might be worth an exception

Reply:
disregard that then <3